lib/Injections/ConsoleAgent.js

```diff
@@ -54,40 +54,58 @@ module.exports = function injection(require, debug, options) {
     messagesCache = {};
   }
 
+  var originalConsole = console;
+
   function wrapConsole() {
     if (consoleIsWrapped) return;
-
-    ['log', 'warn', 'error', 'info'].forEach(function(fn) {
-      console['_' + fn] = console[fn];
-      console[fn] = wrapFunction(fn, console[fn]);
-    });
-
-    console.assert = wrapAssertFunction('assert', console.assert);
+    var _console = {};
+    for (var method in originalConsole) {
+      if (['log', 'warn', 'error', 'info'].indexOf(method) > -1) {
+        _console[method] = wrapFunction(method, originalConsole[method]);
+      } else if (method === 'assert') {
+        _console.assert = wrapAssertFunction('assert', originalConsole.assert);
+      } else {
+        _console[method] = originalConsole[method];
+      }
+    }
 
     consoleIsWrapped = true;
+
+    Object.defineProperty(global, 'console', {
+      configurable: true,
+      enumerable: true,
+      get: function() {
+        return _console;
+      }
+    });
   }
 
   function unwrapConsole() {
     if (!consoleIsWrapped) return;
 
     clearMessages();
-    Object.keys(console).forEach(function(fn) {
-      if (!/^_/.test(fn)) {
-        console[fn] = console['_' + fn];
-        delete console['_' + fn];
+    Object.defineProperty(global, 'console', {
+      configurable: true,
+      enumerable: true,
+      get: function() {
+        return originalConsole;
       }
     });
+    // Object.keys(console).forEach(function(fn) {
+    //   if (!/^_/.test(fn)) {
+    //     console[fn] = console['_' + fn];
+    //     delete console['_' + fn];
+    //   }
+    // });
```

Comment:
We usually remove commented-out code, there is no good in keeping it around. It can be always found in `git` history if needed.

Isn't this change breaking the ability of Node Inspector to display `console.log` calls made by the app in the browser UI?